### PR TITLE
Tweak language re -aths files

### PR DESCRIPTION
**Pull request by ageis**
This is intended to address issue #1030 - instead of suggesting that the user needs to add the values to their torrc, it says that the next step ('tails_files/install.sh') will take care of it.

### CODE REVIEW

**Comment by conorsch** [comment]
> As part of the next step

Which step? It's not actually the next one, it's about two ahead. Maybe rephrase more generally, saying it's a later step. 

> this will be added to your Tor configuration file

Consider adding "automatically" to make it clear that the addition does not require manual intervention by the admin—beyond running the script that makes the addition, of course.

**Comment by ageis** [comment]
I'm with you on the second suggestion, but not sure what you mean by two steps ahead. Running install.sh from tails-files is literally in the very next section. I guess later step should be fine though.

**Comment by conorsch** [comment]
The next step mentioned is: 

> Update the inventory, replacing the IP addresses with the corresponding onion addresses from app-ssh-aths and mon-ssh-aths

**Comment by conorsch** [comment]
Thanks; lgtm!

**Comment by garrettr** [comment]
Changes lgtm. Nit: neither commit message is very descriptive. @ageis Could you rebase this into one commit, and make the commit message more descriptive?

**Comment by ageis** [comment]
@garrettr done. Thanks again for showing me how to rebase!

**Comment by garrettr** [comment]
@ageis lgtm, thanks for rebasing it!
